src/components/SortableCommentList.js

```diff
@@ -0,0 +1,207 @@
+    });
+  }
+
+  onPostHearingComment(text, authorName, pluginData, geojson, label, images) { // eslint-disable-line
+    const { postComment /* , section, fetchComments */ } = this.props;
+    const hearingSlug = this.props.hearingSlug;
+    const { authCode } = this.props.location.query;
+    const mainSection = this.props.mainSection;
+    const commentData = { text, authorName, pluginData: null, authCode, geojson: null, label: null, images };
+    postComment(hearingSlug, mainSection.id, commentData);
+    console.log(this.props.sectionComments);
+  }
+
+  onPostSectionComment(sectionId, sectionCommentData) {
+    const {postComment} = this.props;
+    const hearingSlug = this.props.hearingSlug;
+    const {authCode} = this.props.location.query;
+    const commentData = Object.assign({authCode}, sectionCommentData);
+    postComment(hearingSlug, sectionId, commentData);
+  }
+
+  onVoteComment(commentId, sectionId) {
+    const {postVote} = this.props;
+    const hearingSlug = this.props.hearingSlug;
+    postVote(commentId, hearingSlug, sectionId);
+  }
+
+  handleReachBottom() {
+    const {sectionComments} = this.props;
+    if (sectionComments && sectionComments.count !== sectionComments.results.length) {
+      setTimeout(() => this.fetchMoreComments(), 1000);
+      this.setState({showLoader: true});
+    }
+  }
+
+  render() {
+    const {
+      displayVisualization,
+      fetchComments,
+      // postComment,
+      section,
+      sectionComments,
+      canVote,
+      // voteComment,
+      ...rest
+    } = this.props;
+
+    const showCommentList = section && sectionComments && get(sectionComments, 'results');
+    return (
+      <div className="sortable-comment-list">
+        <h2><FormattedMessage id="comments"/>
+          <div className="commenticon">
+            <Icon name="comment-o"/>&nbsp;{get(sectionComments, 'count') ? sectionComments.count : ''}
+          </div>
+        </h2>
+        {
+          (typeof window !== 'undefined') && showCommentList && displayVisualization && get(section, 'plugin_identifier') === 'mapdon-ksv' ?
+            <div className="comments-visualization">
+              <MapdonKSVPlugin
+                data={section.plugin_data}
+                pluginPurpose="viewComments"
+                comments={sectionComments.results}
+              />
+              <div className="image-caption">Kaikki annetut kommentit sekä siirretyt ja lisätyt asemat kartalla.</div>
+              <MapdonKSVPlugin
+                data={section.plugin_data}
+                pluginPurpose="viewHeatmap"
+                comments={sectionComments.results}
+              />
+              <div className="image-caption">Siirrettyjen ja lisättyjen asemien tiheyskartta.</div>
+            </div>
+            : null
+        }
+        <form className="sort-selector">
+          <FormGroup controlId="sort-select">
+            <ControlLabel><FormattedMessage id="commentOrder"/></ControlLabel>
```

Comment:
I would move the ordering control to the right hand side (and make it narrower) so it is more in line with location of the ordering control in the hearing search page.

Also, this allows the ordering control to be on the same row as the Answer button when the comment input box is closed. This means less vertical whitespace and more comments visible for the user.